Food.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html style="font-size: 16px;">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="utf-8">
+    <meta name="keywords" content="">
+    <meta name="description" content="">
+    <meta name="page_type" content="np-template-header-footer-from-plugin">
+    <title>Food</title>
+    <link rel="stylesheet" href="main.css" media="screen">
+<link rel="stylesheet" href="Food.css" media="screen">
+    <script class="u-script" type="text/javascript" src="jquery.js" defer=""></script>
+    <script class="u-script" type="text/javascript" src="main.js" defer=""></script>
+    
+    <link id="u-theme-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i|Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i">
+    
+    <script type="application/ld+json">{
+		"@context": "http://schema.org",
+		"@type": "Organization",
+		"name": "Wayne",
+		"url": "index.html",
+		"logo": "images/360_F_359190084_yNAOJ5FrCbNHklodaobTM2jsdHEb3mqs.jpg"
+}</script>
```

Comment:
would it not be better to put the JavaScript in a separate file

Reply:
I agree with erahman, next time let's use external javascript to make the code more readable